doc/source/tune/_tutorials/tune-pytorch-cifar.rst

```diff
@@ -95,13 +95,43 @@ We also split the training data into a training and validation subset. We thus t
 80% of the data and calculate the validation loss on the remaining 20%. The batch sizes
 with which we iterate through the training and test sets are configurable as well.
 
+Adding (multi) GPU support with DataParallel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Image classification benefits largely from GPUs. Luckily, we can continue to use
+PyTorch's abstractions in Ray Tune. Thus, we can wrap our model in ``nn.DataParallel``
+to support data parallel training on multiple GPUs:
+
+.. code-block:: python
+
+    device = "cpu"
+    if torch.cuda.is_available():
+        device = "cuda:0"
+        if torch.cuda.device_count() > 1:
+            net = nn.DataParallel(net)
+    net.to(device)
+
+By using a ``device`` variable we make sure that training also works when we have
+no GPUs available. PyTorch requires us to send our data to the GPU memory explicitly,
+like this:
+
+.. code-block:: python
+
+    for i, data in enumerate(trainloader, 0):
```

Comment:
pro tip: enumerate doesn't need 2 args

Reply:
Haha, I know! I actually copied that directly from the PT tutorial code (https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#train-the-network) and wanted to alter as few as possible, so I'd like to keep it that way